### PR TITLE
Maddie tests

### PR DESCRIPTION
**Pull request by Prtfw**
tests for reducers and epics

### CODE REVIEW

File: src/epics/index.js [hunk 1]
```diff
@@ -22,8 +22,9 @@ export const apiFetchEpic = (action$, blah, { ajax }) =>
   action$
     .ofType(GET_API_RESULTS)
     .mergeMap(action => ajax(`http://lcboapi.com/products?q=${action.payload}&access_key=${LCBO_API_KEY}`)) //TODO: use search string later.
-    .map(res => {
-      return sendAPIResults(res)});
+    .map(({response}) => {
+      // console.log(26, response);
```
Comment (jocelynjeffrey) [inline src/epics/index.js]: You can take out the console.log here

**Review by jocelynjeffrey — APPROVED** [review]
A console.log you can remove (in comments) but all the tests are passing. Looks good!

File: src/epics/index.js [hunk 2]
```diff
@@ -16,11 +18,13 @@ export const fetchProducts = (searchString = '') => {
 }
 
 
-const apiFetchEpic = action$ =>
+export const apiFetchEpic = (action$, blah, { ajax }) =>
```
Comment (jprittie) [inline src/epics/index.js]: Okay if we replace 'blah' with _, ?

File: src/tests/epics/epics.test.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+
+    apiFetchEpic(action$, null, { ajax })
+      .subscribe(result => {
+        expect(result.type).toBe(SEND_API_RESULTS);
+        expect(result.payload).toEqual(['socks', 'mocks']);
+
+        done();
+      })
+
```
Comment (astr0-4) [inline src/tests/epics/epics.test.js]: 💯  wooooo!

**Review by astr0-4 — APPROVED** [review]
very nice tests!! 👌 

Only thing I would like to see is some formatting, which prettier would take care of.

**Comment by Prtfw** [comment]
thx for the review folks, changes made.... time to mergy merge

**Pull request merged**